Source/santad/EventProviders/SNTDeviceManager.h

```diff
@@ -0,0 +1,45 @@
+/// Copyright 2021 Google Inc. All rights reserved.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///    http://www.apache.org/licenses/LICENSE-2.0
+///
+///    Unless required by applicable law or agreed to in writing, software
+///    distributed under the License is distributed on an "AS IS" BASIS,
+///    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+///    See the License for the specific language governing permissions and
+///    limitations under the License.
+#import <DiskArbitration/DiskArbitration.h>
+#import <Foundation/Foundation.h>
+
+#include <EndpointSecurity/EndpointSecurity.h>
+
+@interface SNTDeviceManager : NSObject
+
+@property BOOL subscribed;
+@property BOOL blockUSBMount;
+@property NSArray<NSString *> *remountArgs;
+
+- (instancetype)init;
+
+- (void)listen;
+- (void)listenDA;
+- (void)listenES API_AVAILABLE(macos(10.15));
+
+- (void)flushCache API_AVAILABLE(macos(10.15));
+- (void)remount:(DADiskRef)disk mountMode:(long)remountMask;
+
+// handleESMessage handles an ES message synchronously. This will block all incoming ES events
+// until either we serve a response or we hit the auth deadline. Prefer [SNTDeviceManager
+// handleESMessageWithTimeout]
+- (void)handleESMessage:(const es_message_t *)m
+             withClient:(es_client_t *)c API_AVAILABLE(macos(10.15));
+
+- (void)handleESMessageWithTimeout:(const es_message_t *)m
```

Comment:
It doesn't look like the `handleESMessage*` methods are called outside of this class. Recommend not exposing externally.

Reply:
Done